content/g-sgov/g-sgov-věda-a-výzkum/g-sgov-věda-a-výzkum-glosář.ttl

```diff
@@ -175,11 +163,29 @@ g-sgov-věda-a-výzkum-pojem:má-kontakt-pro-přihlášení a skos:Concept;
   skos:prefLabel "má kontakt pro přihlášení"@cs;
   skos:scopeNote ""@cs .
 
+g-sgov-věda-a-výzkum-pojem:orjk a skos:Concept;
+  skos:definition "Kód organizační jednotky (ORJK), který se využívá např. v Informačním systém výzkumu, experimentálního vývoje a inovací (ISVaV) a ve statistikách MŠMT. Používá se pro identifikaci jednotek, které nemají IČO, např. fakulty v rámci univerzity."@cs;
+  skos:inScheme g-sgov-věda-a-výzkum:glosář;
+  skos:prefLabel "orjk"@cs .
```

Comment:
Čekal bych "Kód organizační jednotky" jako preflabel a "ORJK" jako altlabel.

Reply:
Já na tohle vůbec nesahal... jediný co jsem přidával bylo "g-sgov-věda-a-výzkum-pojem:má-termín-přihlášení" 🤷🏻‍♂️

Reply:
Jj, ten diff je debilní a blbě se to poznává. Ale otázka platí (mimo tento PR) :-), jen je možná pro @martinnec .